plugin-server/src/worker/ingestion/event-pipeline/runner.ts

```diff
@@ -120,49 +120,76 @@ export class EventPipelineRunner {
             this.poEEmbraceJoin = true
         }
 
-        let processPerson = true
-        if (event.properties && event.properties.$process_person === false) {
-            // We are purposefully being very explicit here. The `$process_person` property *must*
-            // exist and be set to `false` (not missing, or null, or any other value) to disable
-            // person processing.
-            processPerson = false
-
-            if (['$identify', '$create_alias', '$merge_dangerously', '$groupidentify'].includes(event.event)) {
-                const warningAck = captureIngestionWarning(
+        const kafkaAcks: Promise<void>[] = []
```

Comment:
maybe rename this to postponedAwaits so we can use it for anything else we await on, e.g. person props PG update stuff

Reply:
They're even called `ackPromises` up higher by the caller: https://gitkraken.dev/link/dnNjb2RlOi8vZWFtb2Rpby5naXRsZW5zL2xpbmsvci83MWVhZjdlMWQzMWFjNzUxOGYyMjcyMDMzZmFlMWI1N2QyODExNTQwP3VybD1naXQlNDBnaXRodWIuY29tJTNBUG9zdEhvZyUyRnBvc3Rob2cuZ2l0?origin=gitlens

If it's OK, I think we should rename when we change them to include other things.

Reply:
totally, just a "maybe" ;)